buildtools/windows_inno_installer.iss

```diff
@@ -56,7 +56,7 @@ AppPublisher={#MyAppPublisher}
 AppPublisherURL={#MyAppURL}
 AppSupportURL={#MyAppSuppURL}
 AppUpdatesURL={#MyAppURL}
-DefaultDirName={userdocs}\..\.tari-testnet
+DefaultDirName={userdocs}\..
```

Comment:
If the default install folder name needs to be changed to something else, only a single line needs to be changed, for example, change
```
DefaultDirName={userdocs}\..\.tari-testnet
```
to
```
DefaultDirName={userdocs}\..\tari
```
or to
```
DefaultDirName={userdocs}\..\tari-testnet
```
All other changes are unnecessary.

Reply:
Fair point, will revert.

Reply:
Just a note, ```.file``` or ```.folder``` in Unix is a hidden file or folder and is meant to be like a hidden settings folder. I think the Windows folder name is just following from the Nix developement. If we keeping with standard folder structure, we should use the same for Windows/OSX/Linux or else we need a doc that shows the mapping for each supported OS?